tailscale/resource_tailnet_key.go

```diff
@@ -27,6 +27,15 @@ func resourceTailnetKey() *schema.Resource {
 				Description: "Indicates if the key is ephemeral.",
 				ForceNew:    true,
 			},
+			"tags": {
+				Type: schema.TypeList,
```

Comment:
I think this should probably be `TypeSet` instead of `TypeList`, so that duplicate tags aren't allowed.

Reply:
Nice catch, done